src/rust/receiver-mock/src/main.rs

```diff
@@ -58,6 +58,13 @@ async fn main() -> std::io::Result<()> {
           .help("Use to print received metrics (with dimensions) on stdout")
           .takes_value(false)
           .required(false))
+      .arg(Arg::with_name("store_metrics")
+          .short("s")
+          .long("store-metrics")
+          .value_name("store_metrics")
+          .help("Use to store metrics which will then be returned via /metrics-samples")
+          .takes_value(false)
+          .required(false))
```

Comment:
I'd say let's keep this disabled by default because people might want to use receiver-mock in performance heavy scenarios and this might be impeding the performance.

Reply:
I don't know if it's a good use of our time to care about this in the context of an internal tool only used by <10 people for testing. I don't mind merging this as-is, since you've already made the effort, but I have a change in the works that adds log storage, and I don't want to spend time making it optional.

Reply:
I disagree. It's rather simple to add that flag for now whereas an impact on the performance/memory requirements of receiver mock might be significant.

Reply:
I'm not going to hold this PR up with this discussion, we can continue here: https://github.com/SumoLogic/sumologic-kubernetes-tools/pulls where I pointedly don't make log storage configurable. 😉